src/fides/api/models/detection_discovery.py

```diff
@@ -278,6 +280,9 @@ class StagedResource(Base):
         default=dict,
     )
 
+    # hidden flag, used by some parts of the dataset lifecycle experience
+    hidden = Column(Boolean, default=False, nullable=False)
```

Comment:
additional comment - @thingscouldbeworse how does hidden differ from the existing 'ignored' status of a staged resource? in the mockups i'd assumed that this was just a cosmetic change to change how we represented 'ignored' resources, but maybe i'm not understanding that well...

Reply:
After the slack conversation we have only `data_uses` being added to the `stagedresource` table, no `hidden`: https://github.com/ethyca/fides/pull/5600/files#diff-7beb05510dec5a679bfe204be998b2a29ef2f14c216249e385bd6d1500ba4437R21